e2e/main_test.go

```diff
@@ -20,11 +21,9 @@ var (
 	client          kubernetes.Interface
 	sshClient       *SSHClient
 	expectedMasters int // hint for tests to figure out how to fail or block on resources missing
+	namespace       = fmt.Sprintf("bootkube-e2e-%x", rand.Int31())
```

Comment:
Note: this was mostly to make local testing easier. Otherwise I had to wait a minute or two before the apiserver finished terminating the namespace from the previous run.